statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -28,6 +31,31 @@
 import warnings
 
 
+def _handle_args(names, defaults, *args, **kwargs):
```

Comment:
Notwithstanding @josef-pkt 's admonition against using `inspect`, couldn't this function be simplified a bunch using `inspect.getcallargs`?

Reply:
What I don't understand is why scipy's `minimize` function doesn't allow `kwargs` specification.